blockchain/indexers/manager.go

```diff
@@ -301,16 +301,16 @@ func (m *Manager) Init(chain *blockchain.BlockChain, interrupt <-chan struct{})
 			return err
 		}
 
+		// Nothing to do if the index does not have any entries yet.
 		switch v := indexer.(type) {
 		case *SlpIndex:
 			if height == v.config.StartHeight {
 				continue
 			}
-		}
-
-		// Nothing to do if the index does not have any entries yet.
-		if height == -1 {
-			continue
+		default:
```

Comment:
Why does this need to be moved?

Reply:
It doesn't need to move, it just looked cleaner inside the switch because it is the same check basically.

Reply:
Ahh that makes sense!

Reply:
Since I've realized the check needs to be moved to the inner for loop (in 81d462c), and so this switch statement is gone now.